src/components/configuration/DataLakeAlertsConfig.vue

```diff
@@ -0,0 +1,622 @@
+                    >
+                      <v-icon size="18" icon="mdi-delete" />
+                    </button>
+                  </div>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+
+        <!-- Empty State -->
+        <div v-else class="text-center py-6 text-slate-400">
+          <v-icon size="48" icon="mdi-bell-outline" class="mb-2 opacity-50" />
+          <p class="text-sm">No alerts configured yet.</p>
+        </div>
+
+        <!-- Add New Alert Button -->
+        <button
+          class="w-full py-2 px-4 rounded bg-[#FFFFFF11] hover:bg-[#FFFFFF22] transition-colors text-sm font-medium flex items-center justify-center gap-2"
+          @click="openCreateDialog"
+        >
+          <v-icon size="18" icon="mdi-plus" />
+          Add New Alert
+        </button>
+      </div>
+    </template>
+  </ExpansiblePanel>
+
+  <!-- Create/Edit Alert Dialog -->
+  <v-dialog v-model="isDialogOpen" max-width="650px">
+    <v-card class="rounded-lg" :style="interfaceStore.globalGlassMenuStyles">
+      <v-card-title class="text-h6 font-weight-bold py-4 text-center">
+        {{ isEditing ? 'Edit Alert' : 'New Alert' }}
+      </v-card-title>
+      <v-card-text class="px-6">
+        <div class="flex flex-col gap-4">
+          <!-- Alert Name -->
+          <v-text-field
+            v-model="editingAlert.name"
+            label="Alert Name"
+            variant="outlined"
+            density="compact"
+            hide-details
+            placeholder="e.g., High CPU Temperature"
+          />
+
+          <!-- Conditions Section -->
+          <div class="flex flex-col gap-3">
+            <div class="flex items-center justify-between">
+              <label class="text-sm font-medium">Conditions (all must match)</label>
+              <button
+                class="text-xs px-2 py-1 rounded bg-[#FFFFFF22] hover:bg-[#FFFFFF33] transition-colors"
+                @click="addCondition"
+              >
+                + Add Condition
+              </button>
+            </div>
+
+            <div
+              v-for="(condition, index) in editingAlert.conditions"
+              :key="index"
+              class="flex flex-col gap-2 p-3 rounded bg-[#FFFFFF11]"
+            >
+              <!-- Variable Selection for this condition -->
+              <div class="flex items-center gap-2">
+                <v-autocomplete
+                  v-model="condition.variableId"
+                  :items="availableVariables"
+                  item-title="label"
+                  item-value="value"
+                  label="Variable"
+                  variant="outlined"
+                  density="compact"
+                  hide-details
+                  class="flex-1"
+                  @update:model-value="onConditionVariableChanged(condition)"
+                />
+                <v-tooltip text="Copy variable name for use in custom message" location="top">
```

Comment:
Uhh, aren't data lake references supposed to be made using the ID, not the name?

Variable names aren't guaranteed to be unique...